### ISSUE TITLE
plotting training loss per epoch 

### ISSUE THREAD

**Issue by sergioconsoli** [awslabs/gluon-ts#659]
Hi,
I'm using DeepAREstimator for forecasting of a time series.
When I lunch the training on my estimator:
estimator.train(training_data=train_ds)
I see logs about some information per epoch, like
Elapsed time per epoch, Evaluation metric 'epoch_loss'=.., Learning rate per epoch...
These are useful information and for example I would like to use the loss per epoch to make a plot at the end of my training.
But how can I retrieve these information when the training has finished?
They are showed on log during training but not stored in any variable.

Can you help me please?

Best,
Sergio
